core/src/test/java/com/exadel/etoolbox/linkinspector/core/servlets/ReplaceByPatternServletTest.java

```diff
@@ -497,14 +494,22 @@ private int countLines(String str) {
 
     private String[] loadSelectedPropertiesValues() throws IOException {
         ClassLoader classLoader = getClass().getClassLoader();
-        File file = new File(classLoader.getResource(TEST_SELECTED_VALUES_PATH).getFile());
+        java.net.URL resourceUrl = classLoader.getResource(TEST_SELECTED_VALUES_PATH);
```

Comment:
(cosmetic) Import the `URL` class via standard import. you don't need `java.net.`

Reply:
done